scripts/fiber/tests-failing.txt

```diff
@@ -37,6 +37,9 @@ src/renderers/dom/shared/__tests__/ReactRenderDocument-test.js
 * should throw on full document render w/ no markup
 * supports findDOMNode on full-page components
 
+src/renderers/dom/shared/wrappers/__tests__/ReactDOMInput-test.js
+* should control a value in reentrant events
```

Comment:
Ah yes. I missed this too. This is probably either a bug or behavior change regarding event timing.